### ISSUE TITLE
Crash application with ember 2.11

### ISSUE THREAD

**Issue by ValentinTrinque** [eliksir/ember-progress#2]
When trying to run the app, with this package, it crashes :

```
➜ ember s
bootstrap-sassy config: some JS loaded [dropdown,modal], glyphicons disabled
    Cannot find module 'my_project/bower_components/ember/ember-template-compiler.js'
    Error: Cannot find module 'my_project/bower_components/ember/ember-template-compiler.js'
        at Function.Module._resolveFilename (module.js:470:15)
        at Function.Module._load (module.js:418:25)
        at Module.require (module.js:498:17)
        at require (internal/module.js:20:19)
        at Class.htmlbarsOptions (my_project/node_modules/ember-progress/node_modules/ember-cli-htmlbars/ember-addon-main.js:96:25)
        at Object.toTree (my_project/node_modules/ember-progress/node_modules/ember-cli-htmlbars/ember-addon-main.js:30:43)
        at my_project/node_modules/ember-cli-preprocess-registry/preprocessors.js:180:26
        at Array.forEach (native)
        at processPlugins (my_project/node_modules/ember-cli-preprocess-registry/preprocessors.js:178:11)
        at module.exports.preprocessTemplates (my_project/node_modules/ember-cli-preprocess-registry/preprocessors.js:161:10)
```

**Versions :**

- Ember : 2.11.2
- Ember Data : 2.11.1
- jQuery : 3.1.1

**Comment by simenheg** [comment]
Thanks for the report, @ValentinTrinque. Could you test again with the latest master?

**Comment by ValentinTrinque** [comment]
This doesn't seem to work. I still experience the issue.

**Comment by simenheg** [comment]
OK, it seems that the ember-cli-htmlbars package is causing the crash. Please try again now.

**Comment by ValentinTrinque** [comment]
It now works as expected ! :) Thanks for your quick fix

**Comment by ValentinTrinque** [comment]
Do you plan to release a new version today ?

**Comment by simenheg** [comment]
Good. Thanks for testing, @ValentinTrinque. I've released version 0.2.1, which should be available from npm now. :)
